.github/workflows/build_documentation.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     with:
       commit_sha: ${{ github.sha }}
       package: peft
-      notebook_folder: peft_docs
+      notebook_folder: peft_doc
```

Comment:
Ok for me to try this to remain consistent with the other libraries.

However, I'm not sure if this'll fix it. For example, Diffusers is also set up to `diffusers_doc` but when you click on a `Open in Colab` button you'll get the same error because either the doc wasn't converted to a notebook or it failed to push it to the https://github.com/huggingface/notebooks repo.

cc @mishig25 @coyotte508

Reply:
I'm definitely not 100% sure if it will fix the issue or not :-) I've just been trying to understand all the github CI/CD code and this is what I would try. Perfectly fine to revert this commit if it doesn't fix the issue of course.

Reply:
@stevhliu could you maybe give an example of such a diffuser documentation page which has a broken link to the google Colab? I tried some random ones and they worked for me. If you could provide an example broken link, that would be great.

Reply:
Sure, thanks for trying to troubleshoot the issue as well! ❤️ 

* this [Inference with PEFT](https://colab.research.google.com/github/huggingface/notebooks/blob/main/diffusers_doc/en/pytorch/using_peft_for_inference.ipynb) doc from Diffusers was added ~5 months ago
* this [Generation with LLMs](https://colab.research.google.com/github/huggingface/notebooks/blob/main/transformers_doc/en/pytorch/llm_tutorial.ipynb) tutorial from Transformers was added ~7 months ago